packages/@vue/cli/bin/vue.js

```diff
@@ -57,6 +57,9 @@ program
   .option('-b, --bare', 'Scaffold project without beginner instructions')
   .action((name, cmd) => {
     const options = cleanArgs(cmd)
+    if (program.args.length > 1){
```

Comment:
There're quite a few false positives with this conditional expression actually.
It might be helpful to write a test case for it so that the problem could be identified earlier.

Reply:
@sodatea So you are expecting me to write separate test case right within the `scripts/test.js` file?

Reply:
@jamesgeorge007 
Ah, I forgot we didn't have a test setup for cli utilities…
So you can skip it if you feel it's too tedious for such a small patch.

But the false positives have to be fixed. Your code would log warning even for commands like`vue create a` but not for `vue create -d a b`